### PR TITLE
Merge latest translations

### PR DESCRIPTION
**Pull request by Cervator**
Fetching latest from Weblate before #2279 as it touches a language file. Also looks like there are conflicts to resolve.

### CODE REVIEW

**Comment by GooeyHub** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.terasology.org/job/TerasologyPRs/580/
Hooray Jenkins reported success with all tests good!

**Comment by Cervator** [comment]
@Trekmarvel @Translator5 @rzats thanks for the translations! Could you confirm that they all made it OK? I had to commit one file manually and resolve some conflicts.

Should be able to check in Weblate shortly or just verify in the commits here and/or https://github.com/MovingBlocks/Terasology/commit/9a39a4497b05b220b56afbf7ee04b72cb5a7df85

**Comment by Cervator** [comment]
Caught one spot where the merge reverted a small legit change: https://github.com/MovingBlocks/Terasology/commit/4d44f40237033ab8c518dfa68716f23239befb61 - at least I think legit!

File: engine/src/main/resources/assets/i18n/menu_uk.lang [hunk 1]
```diff
@@ -115,10 +115,10 @@
     "video-preset": "Передустановка", 
     "video-preset-custom": "Користувацькі", 
     "video-preset-epic": "Епічні", 
-    "video-preset-insane": "Шалені",
+    "video-preset-insane": "Шалені",
```
Comment (rzats) [inline engine/src/main/resources/assets/i18n/menu_uk.lang]: @Cervator - Ukrainian looks good! Random whitespace change here, but that's probably just a Weblate formatting quirk.

**Comment by Translator5** [comment]
German looks also good. Sorry for that while.

**Pull request merged**